pkg/arvo/app/dojo.hoon

```diff
@@ -1349,23 +1350,40 @@
   !>(state)
 ::
 ++  on-load
-  |=  =old-state=vase
-  =/  old-state  !<(house old-state-vase)
-  `..on-init(state old-state)
+  |=  old=vase
+  ?:  ?=(%6 +<.old)
+    `..on-init(state !<(house old))
+  =/  old-5  !<([%5 egg=@u hoc=(map id session)] old)
+  =/  =house  [%6 egg.old-5 hoc.old-5 *(set ship)]
+  `..on-init(state house)
 ::
 ++  on-poke
   |=  [=mark =vase]
   ^-  (quip card:agent:gall _..on-init)
   =^  moves  state
     ^-  (quip card:agent:gall house)
     ?+  mark  ~|([%dojo-poke-bad-mark mark] !!)
+    ::
         %sole-action
       =/  act  !<(sole-action vase)
       he-abet:(~(he-type he hid id.act ~ (~(got by hoc) id.act)) act)
     ::
         %lens-command
       =+  !<([=id =command:lens] vase)
       he-abet:(~(he-lens he hid id ~ (~(got by hoc) id)) command)
+    ::
+        %grant-access
+      =/  who  !<(@p vase)
+      `state(acl (~(put in acl) who))
+    ::
+        %revoke-access
+      =/  who  !<(@p vase)
+      :_  state(acl (~(del in acl) who))
+      [%give %kick ~ `who]~
+    ::
+        %list-access
+      ~&  acl
+      `state
     ::
         %json
       ~&  jon=!<(json vase)
```

Comment:
btw, should this `%json` mark be removed? I don't see what it would be useful for.

Reply:
I suspect this was added for Soto but ultimately %sole-action was used directly.  @matildepark Do you know if that's true?

Reply:
We didn't touch dojo.hoon for Soto, as I remember it.

Reply:
The blames on that line go back a-ways for lens, talk, dojo conversions for mall.

Reply:
Ah, lens makes sense.

Yes, pretty sure this can just be removed.